elements/research/randload.cc

```diff
@@ -0,0 +1,56 @@
+/*
+ * RandLoad.{cc,hh}
+ */
+
+#include "randload.hh"
+
+#include <click/config.h>
+#include <click/glue.hh>
+#include <click/args.hh>
+
+CLICK_DECLS
+
+RandLoad::RandLoad() {
```

Comment:
Should all brackets go to the next line as in the configure method?

Reply:
I think the Click style is like this. The rest should change. But it's already a mess anyway...